agg-ast/schema_derivation/src/schema_derivation.rs

```diff
@@ -725,11 +823,11 @@ impl DeriveSchema for TaggedOperator {
                 state,
                 Schema::Atomic(Atomic::Double),
             ),
```

Comment:
This seems to have been a mistake, $percentile always returns arrays in all contexts